CHANGELOG_PENDING.md

```diff
@@ -45,6 +45,8 @@
     use `make build_c` / `make install_c` (full instructions can be found at
     https://tendermint.com/docs/introduction/install.html#compile-with-cleveldb-support)
   * Use `boltdb` tag to compile Tendermint with bolt db
+- [p2p] [\#3666](https://github.com/tendermint/tendermint/issues/3666)
```

Comment:
No need to change anything but just FYI: you do not need to add the link yourself here. There is a little python script that generates the link to the issue from `[\#3666]`.